src/server/managers/AppListenerManager.ts

```diff
@@ -52,7 +53,7 @@ export class AppListenerManager {
     }
 
     // tslint:disable-next-line
-    public async executeListener(int: AppInterface, data: IMessage | IRoom | IUser): Promise<void | boolean | IMessage | IRoom | IUser> {
+    public async executeListener(int: AppInterface, data: IMessage | IRoom | IUser | ILivechatRoom): Promise<void | boolean | IMessage | IRoom | IUser | ILivechatRoom> {
```

Comment:
This is getting kinda hard to manage... In the _rich messages_ we have added one type here as well, and this tends to keep growing. We should think of a way of refactoring this in the future